### ISSUE TITLE
Training log update

### ISSUE THREAD

**Issue by peiyingxin** [yzd-v/FGD#48]
thanks for sharing your excellent job!
can you update your training logs? when I have runned
configs/distillers/fgd/fgd_retina_rx101_64x4d_distill_retina_r50_fpn_2x_coco.py , I only got 30.4 mAP
Epoch(val) [24][625]   bbox_mAP: 0.3040, bbox_mAP_50: 0.4980, bbox_mAP_75: 0.3140, bbox_mAP_s: 0.1640, bbox_mAP_m: 0.3300, bbox_mAP_l: 0.4160, bbox_mAP_copypaste: 0.304 0.498 0.314 0.164 0.330 0.416

I don't konw why
thank your~

**Comment by yzd-v** [comment]
The log cannot help you to solve the problem. There must be something wrong with the codes. What's your mmdet version?

**Comment by peiyingxin** [comment]
> The log cannot help you to solve the problem. There must be something wrong with the codes. What's your mmdet version?

mmdet==2.25.0 mmcv-full==1.6.1

**Comment by yzd-v** [comment]
Please refer README for the higher mmdet version. You might initialize the student and teacher wrongly.

**Comment by peiyingxin** [comment]
In my training log, I found
2022-08-05 17:52:24,038 - mmdet - INFO - Set random seed to 0, deterministic: False
2022-08-05 17:52:26,558 - mmdet - INFO - initialize ResNeXt with init_cfg {'type': 'Pretrained', 'checkpoint': 'open-mmlab://resnext101_64x4d'}
2022-08-05 17:52:27,039 - mmdet - INFO - initialize FPN with init_cfg {'type': 'Xavier', 'layer': 'Conv2d', 'distribution': 'uniform'}
2022-08-05 17:52:27,095 - mmdet - INFO - initialize RetinaHead with init_cfg {'type': 'Normal', 'layer': 'Conv2d', 'std': 0.01, 'override': {'type': 'Normal', 'name': 'retina_cls', 'std': 0.01, 'bias_prob': 0.01}}
2022-08-05 17:52:27,234 - mmdet - INFO - initialize ResNet with init_cfg {'type': 'Pretrained', 'checkpoint': 'torchvision://resnet50'}
2022-08-05 17:52:27,452 - mmdet - INFO - initialize FPN with init_cfg {'type': 'Xavier', 'layer': 'Conv2d', 'distribution': 'uniform'}
2022-08-05 17:52:27,509 - mmdet - INFO - initialize RetinaHead with init_cfg {'type': 'Normal', 'layer': 'Conv2d', 'std': 0.01, 'override': {'type': 'Normal', 'name': 'retina_cls', 'std': 0.01, 'bias_prob': 0.01}}
It seems that student and teacher all init from pretrained model, right?

**Comment by yzd-v** [comment]
For mmdet>2.12, you have to change the tools/train.py and mmdet/distillation/distillers/detection_distiller.py.  You can refer MGD to change this.

**Comment by peiyingxin** [comment]
OK, I will change code refer MGD.
Thanks for your reply!!
